src/firefish/firefish.rs

```diff
@@ -1318,6 +1318,20 @@ impl megalodon::Megalodon for Firefish {
         ))
     }
 
+    async fn get_status_source(&self, id: String) -> Result<Response<MegalodonEntities::StatusSource>, Error> {
```

Comment:
@h3poteto , are we Ok to use the entity directly from `MegalodonEntities` here?

Reply:
No problem